app/src/main/java/com/hammer/talkbbokki/presentation/navigation/TalkbbokkiNavHost.kt

```diff
@@ -61,7 +61,7 @@ fun TalkbbokkiNavHost(
             navigateToDetail = { level, item ->
                 navController.navigate(
                     DetailDestination.route +
-                        "?level=$level&id=${item.id}&tag=${item.tag}&topic=${item.name}&shareLink=${item.shareLink}"
+                            "?level=$level&id=${item.id}&tag=${item.tag}&topic=${item.name}&shareLink=${item.shareLink}"
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)